internals/cli/ui/ask_test.go

```diff
@@ -244,15 +244,36 @@ func TestChoose(t *testing.T) {
 			in: []string{"\n", "2\n"},
 
 			expected: "bar",
-			out:      "foo? Press [ENTER] for options.\nfoo?\n1) foo\n2) bar\n3) baz\n",
+			out: "foo? Press [ENTER] for options.\n" +
+				"1) foo\n" +
+				"2) bar\n" +
+				"3) baz\n" +
+				"Give number of choice or enter other option: \n",
 		},
 		"choose option of second batch": {
 			question: "foo?",
 
 			in: []string{"\n", "\n", "7\n"},
 
 			expected: "Option 7",
-			out:      "foo? Press [ENTER] for options.\nfoo?\n1) Option 1\n2) Option 2\n3) Option 3\n4) Option 4\n5) Option 5\nPress [ENTER] for more options.\nfoo?\n1) Option 1\n2) Option 2\n3) Option 3\n4) Option 4\n5) Option 5\n6) Option 6\n7) Option 7\n8) Option 8\n9) Option 9\n10) Option 10\nPress [ENTER] for more options.\n",
+			out: "foo? Press [ENTER] for options.\n" +
+				"1) Option 1\n" +
+				"2) Option 2\n" +
+				"3) Option 3\n" +
+				"4) Option 4\n" +
+				"5) Option 5\n" +
+				"Give number of choice or enter other option ([ENTER] for more options): \n" +
```

Comment:
This message is confusing to me. Either I type a number, enter a different region (/answer to foo question) and press ENTER, or I provide an 'empty' answer and just press ENTER to see more options. 

Can we make these use cases a bit more intuitive?

Reply:
What about:
```
Give the number of an option or type a region ([ENTER] for more options):
```